### PR TITLE
Support pod filter in additional Grafana panels

### PR DESCRIPTION
**Pull request by korthout**
## Description

Both the `General > Pod Restarts` and the `General > Process memory
usage` panels did not yet support filtering based on the selected pod.
This was easy to fix, by simply using the `$pod` variable.

## Related issues

No related issue, I just ran into this during my medic duty.

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/zeebe-io/zeebe/compare/stable/0.24...develop?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/0.25`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark 

Documentation: 
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] New content is added to the [release announcement](https://drive.google.com/drive/u/0/folders/1DTIeswnEEq-NggJ25rm2BsDjcCQpDape)

### CODE REVIEW

File: monitor/grafana/zeebe.json [hunk 1]
```diff
@@ -373,7 +373,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "1 - kube_pod_container_status_ready{pod=~\".*\", pod!~\".*curator.*|worker.*|starter.*\", namespace=\"$namespace\"}",
+              "expr": "1 - kube_pod_container_status_ready{pod=~\"$pod\", pod!~\".*curator.*|worker.*|starter.*\", namespace=\"$namespace\"}",
```
Comment (Zelldon) [inline monitor/grafana/zeebe.json:376]: Does it mean that elastic is no longer showed here?

File: monitor/grafana/zeebe.json [hunk 2]
```diff
@@ -1358,14 +1356,14 @@
...
-              "expr": "max(kube_pod_container_resource_limits_memory_bytes{namespace=~\"$namespace\",pod=~\".*zeebe.*\", container=~\"zeebe.*\"})",
+              "expr": "max(kube_pod_container_resource_limits_memory_bytes{namespace=~\"$namespace\",pod=~\"$pod\", container=~\"zeebe.*\"})",
               "hide": false,
               "interval": "",
               "legendFormat": "limit",
               "refId": "B"
             },
             {
-              "expr": "min(kube_pod_container_resource_requests_memory_bytes{namespace=~\"$namespace\",pod=~\".*zeebe.*\", container=~\"zeebe.*\"})",
+              "expr": "min(kube_pod_container_resource_requests_memory_bytes{namespace=~\"$namespace\",pod=~\"$pod\", container=~\"zeebe.*\"})",
```
Comment (Zelldon) [inline monitor/grafana/zeebe.json:1366]: Could you try this on the camunda cloud int stage as well, please. I can remember there were some issues around that but idk what was it, might be the value of the pod variable or something. Please try it out if this works than we can merge it :)

**Review by ChrisKujawa — COMMENTED** [review]
Thanks @korthout :medal_military: Just some small requests please clarify this on INT before we merge it.

File: monitor/grafana/zeebe.json (hunk 1 above)
Comment (korthout) [reply monitor/grafana/zeebe.json:376]: Is still available, but cannot be filtered on as it is not one of the pods.
Screen Shot 2022-01-24 at 10 49 00

File: monitor/grafana/zeebe.json (hunk 2 above)
Comment (korthout) [reply monitor/grafana/zeebe.json:1366]: Tested on https://grafana-worker-1.internal.camunda.io/ where I did not see a difference, except that pod can now be filtered on.

**Comment by korthout** [comment]
@Zelldon Sorry for the super long time between, but this just did not have priority for me at the time. I had some time to look into it now and checked what you asked. Please have another look

**Comment by Zelldon** [comment]
Thanks Nico :+1:

**Review by ChrisKujawa — APPROVED** [review]

**Comment by korthout** [comment]
bors merge

**Pull request merged**